raft/transport_test.go

```diff
@@ -1,6 +1,25 @@
 package raft_test
 
 import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/url"
+	"sync"
+	"testing"
+	"time"
+
+	// "net/http"
```

Comment:
I had to back out all the tests and re-add them one at a time. That's why the commented imports are in there. I need to remove these.